src-docs/src/services/playground/playground.js

```diff
@@ -6,7 +6,7 @@ import { useView, Compiler, Placeholder } from 'react-view';
 import { EuiSpacer, EuiTitle, EuiCodeBlock } from '../../../../src/components';
 import Knobs from './knobs';
 
-export default ({ config, setGhostBackground }) => {
+export default ({ config, setGhostBackground, defaultClasses = [] }) => {
```

Comment:
I would just make this a string. The `classNames` function handles something like `custom-class-1 custom-class-2` appropriately and is more similar to how we normally pass multiple class names. Also, I'd change the name to
```suggestion
export default ({ config, setGhostBackground, playgroundClassName }) => {
```